test/extensions/tracers/opentelemetry/grpc_trace_exporter_test.cc

```diff
@@ -93,25 +93,25 @@ TEST_F(OpenTelemetryGrpcTraceExporterTest, ExportWithRemoteClose) {
           - name: "test"
   )EOF";
 
-  expectStreamMessage(request_yaml);
+  expectTraceExportMessage(request_yaml);
   opentelemetry::proto::collector::trace::v1::ExportTraceServiceRequest request;
   opentelemetry::proto::trace::v1::Span span;
   span.set_name("test");
   *request.add_resource_spans()->add_scope_spans()->add_spans() = span;
   EXPECT_TRUE(exporter.log(request));
 
-  // Close the stream, now that we've created it.
+  // Close the connection, now that we've created it.
```

Comment:
I suggest changing `connection` to `request`. Even with unary gRPC, a connection is shared over multiple RPCs.

Reply:
Thanks for the suggestion. I've updated the comment, but think it might be a good idea to step back and take a second look at the tests before continuing on this, so I'm going to convert this back to a draft.